src/utils/validation.ts

```
@@ -0,0 +1,12 @@
+import { ethers } from 'ethers';
+
+export const isValidEthereumAddress = (address: string | undefined): boolean => {
+    if (!address) return false;
+  
+    try {
+      return ethers.utils.isAddress(address);
+    } catch (e) {
+      //
+    }
```

There's an empty catch block here?